@xen-orchestra/lite/manifest.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "XO Lite",
+  "short_name": "XOLite",
+  "start_url": "/index.html",
+  "display": "standalone",
+  "description": "Integrated local and lightweight solution to manage a local XCP-ng pool.",
+  "icons": [
+    {
+      "src": "/src/assets/favicon.svg",
```

Public files should be in the `public` directory.

`/src/assets/favicon.svg` should be copied to `/public` directory to be used.

```suggestion
      "src": "favicon.svg",
```

---

Have made that change.